profiles/features/wd40/package.use.mask

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# WANG Xuerui <xen0n@gentoo.org> (2022-05-13)
+# dev-lang/rust is not available on ${arch} yet.
+media-gfx/imv svg
+media-libs/libavif rav1e
+
+# Sam James <sam@gentoo.org> (2022-04-22)
+# Needs Rustified librsvg
+>=media-libs/freetype-2.12.0 svg
+media-libs/imlib2 svg
+
+# Michał Górny <mgorny@gentoo.org> (2021-11-26)
+# These package flags require dev-python/cryptography which -- due
+# to the dependency on Rust -- is no longer portable to ${arch}.
+dev-python/twisted conch crypt
```

Comment:
We're not being strictly consistent here, which is okay I guess, but we should note it in the commit message.

twisted[crypt] is solvable on arches without Rust, as long as a non-Rust cryptography version is visible. We only mask Rust versions of cryptography in package.mask.

It's fine to do this in order to cut down on the number of optional / avoidable deps on cryptography given we don't want to have to do it forever, and to minimise our burden - but we should say so explicitly in the mask message as well. If nothing else, to save someone else who is curious like I was.